cmd/helpers.go

```diff
@@ -77,3 +78,20 @@ func VersionStanza() string {
 		Version, Commit, GoVersion, runtime.GOOS, runtime.GOARCH, Date,
 	)
 }
+
+var (
+	BuildInfo = prometheus.NewGaugeFunc(
+		prometheus.GaugeOpts{
+			Name: "theatre_build_info",
+			Help: "A metric with a constant '1' value labeled by version, commit, goversion from which %s was built, and the goos and goarch for the build.",
+			ConstLabels: prometheus.Labels{
+				"version":   Version,
+				"commit":    Commit,
```

Comment:
Won't the commit be adding quite a bit of extra cardinality? How is it different to the version?

Reply:
That would be the commit sha that version has in Git when it was built, so you can refer to the tag and validate it.

Both labels will change at once, so they aren't an issue.

If you wonder how the commit sha is injected, that is via a compile argument via goreleaser. 

```
      -X github.com/gocardless/theatre/cmd.Commit={{.Commit}}
 
```